content/cucumber/step-definitions.md

```diff
@@ -253,3 +253,60 @@ Given("I have {int} {color} balls", function (int1, color) {
 });
 ```
 {{% /block %}}
+
+# How do I group my step definitions?
+
+Avoid writing similar step definitions which can lead to clutter. While documenting your steps helps, making use of **helper methods** to abstract them can do wonders.
+
+For eg, if you have the following steps :
```

Comment:
Please change "eg" -> "example" (for consistency)

Reply:
Much more concise! Very nice :)

Reply:
Sure. And thank you :)